frontend/__tests__/components/modals/column-management-modal.spec.tsx

```diff
@@ -11,71 +11,71 @@ import {
 import { referenceForModel } from '@console/internal/module/k8s';
 import { PodModel } from '@console/internal/models';
 
-const columnManagementID = referenceForModel(PodModel);
+const columnManagementId = referenceForModel(PodModel);
```

Comment:
The rule is `id` all lowercase if it's the first word, but `myID` with uppercase `ID` if a later word of a camel case variable name.

```suggestion
const columnManagementID = referenceForModel(PodModel);
```

Reply:
ok, fixed.